### ISSUE TITLE
Accessibility Issue: Hiding form labels

### ISSUE THREAD

**Issue by RachBreeze** [umbraco/Umbraco.Forms.Issues#1220]
In Umbraco forms it is possible to hide form labels. This is done by setting "show label" on the form label to false:
![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/10153922/380bfdfb-a18d-492a-aed0-d2cabb19366d "Image of Umbraco From Property Editor with show label unchecked")

This removes the label markup from the source:
![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/10153922/a7ff46ed-de6f-41f4-b8f3-aa00614be369 "Image of a view source on Umbraco Form field with no label in the source for an input")
Versus this when label is shown:
![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/10153922/bf3d928f-4ac1-4137-960b-5f7dc2ff66f7 "Image of a view source on Umbraco Form field with a label in the source for an input")

Note the removal of:
`<label for="9a0886b0-0891-4951-f5ab-6b077f2615d0" class="umbraco-forms-label">Search field                                          </label>`

This means the field is not properly identified to screen reader users and means if applied, the form is not considered accessible. From https://www.w3.org/WAI/tutorials/forms/labels:

"A label for a form control helps everyone better understand its purpose. In some cases, the purpose may be clear enough from the context when the content is rendered visually. The label can be hidden visually, though it still needs to be provided within the code to support other forms of presentation and interaction, such as for screen reader and speech input users."

Impacted version:
Umbraco forms v13.1.2 
Umbraco 13.3.0

**Comment by AndyButland** [comment]
Thanks @RachBreeze - and good point.  We'll update this with the next release such that the mark-up remains but the label is visually hidden.
